lnrpc/walletrpc/walletkit_server_test.go

```diff
@@ -0,0 +1,59 @@
+//go:build walletrpc
+// +build walletrpc
+
+package walletrpc
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/lightningnetwork/lnd/input"
+	"github.com/stretchr/testify/require"
+)
+
+// TestWitnessTypeMappings tests that the mapping between the witness type
+// enum defined in the `input` package and the witness type enum protocol
+// buffer definition in the `walletrpc` package is correct.
+func TestWitnessTypeMapping(t *testing.T) {
+	t.Parallel()
+
+	require.Equal(
+		t,
+		len(allWitnessTypes),
+		len(WitnessType_name)-1, // don't count default value
+		"number of witness types should match proto definition",
+	)
+
+	// test that for each individual witness type mapping 1) the mapped
+	// witness types have the same numerical value, and 2) the string
+	// representations are equivalent.
+	for witnessType, witnessTypeProto := range allWitnessTypes {
+		witnessType, ok := witnessType.(input.StandardWitnessType)
+		if !ok {
+			t.Fail()
+		}
+		witnessTypeProto := witnessTypeProto
+
+		t.Run(witnessType.String(), func(tt *testing.T) {
```

Comment:
not an expert on the matter but I think for this small test operation I found the parallel testing a bit overkill, parallel testing can also come with a cost but happy to here your perspective ?

Reply:
hmm I also don't know much beyond "it makes tests run in parallell" lol. I added it in response to a lint. 

[Searching across the codebase](https://github.com/search?q=repo%3Alightningnetwork%2Flnd+%22nolint%3Aparalleltest%22&type=code) it seems that the norm is to not skip the `paralleltest` lint. So I just followed that convention.

Reply:
alright, seems like this is the right approach then.